terraform/install.sh

```diff
@@ -150,15 +215,15 @@ installTerraform
 #export GOOGLE_APPLICATION_CREDENTIALS=""
 #gcloud auth application-default login
 
-# Ensure no google.com accounts early - they are not supported!
 acct=$(gcloud info --format="value(config.account)")
 if [[ $acct == *"google.com"* ]];
 then
-  log "Google.com accounts are currently not supported by Stackdriver Sandbox.";
-  exit;
+  folder_id="262044416022"            
 fi;
```

Comment:
I don't really like having this logic out here on its own. I think this should be moved into createProject, maybe like this:

```
acct=$(gcloud info --format="value(config.account)")
if [ $acct == *"google.com"* ]; then
  gcloud projects create "$project_id" --name="Stackdriver Sandbox Demo" --folder=262044416022
else
  gcloud projects create "$project_id" --name="Stackdriver Sandbox Demo"
fi;
```

Reply:
Additionally, it would be a good idea to prefix Googler-created sandboxes with the user's ldap, so we can keep track of ownership:

```
ldap=$(echo $acct |  cut -f1 -d"@")
project_id=$ldap-$project_id
gcloud projects create "$project_id" --name="Stackdriver Sandbox Demo" --folder=262044416022
```

Do you think something like that is doable?

Reply:
Ok.

Reply:
I don't think we can add idap. The id will exceed its maximum length.